packages/wrangler/src/config-cache.ts

```diff
@@ -0,0 +1,44 @@
+import { mkdirSync, readFileSync, writeFileSync } from "fs";
+import { dirname, join } from "path";
+import { cwd } from "process";
+
+interface ConfigCache {
+  account_id?: string;
+  pages_project_name?: string;
+}
+
+const relativeCacheLocation = join(
```

Comment:
NIT - there is no need to use a `join()` here:

```suggestion
const relativeCacheLocation = join("node_modules/.cache/wrangler-pages/config-cache.json");
```